en/getting_started/ros.md

```diff
@@ -0,0 +1,32 @@
+# Using Generated Source Files with ROS and PX4
+
+If you want to add MAVlink messages or dialects while working with ROS, you need to:
+
+- follow the [instructions](https://github.com/mavlink/mavros/blob/master/mavros/README.md#source-installation) to install mavros from sources and the mavlink-gbp-release which is the mavlink library released for ROS.
```

Comment:
Use the proper name - I don't know what that is MAVROS, mavros? If it is mavros, suggest you do as *mavros* everywhere.